### PR TITLE
fix: nil pointer dereference when failed to scan out the quota records from redis

### PR DESCRIPTION
**Pull request by microyahoo**
Signed-off-by: Liang Zheng <zhengliang0901@gmail.com>

# Comprehensive Summary of your change

# Issue being fixed
Fixes [21045](https://github.com/goharbor/harbor/issues/21045)

Please indicate you've done the following:
- [x] Well Written Title and Summary of the PR
- [x] Label the PR as needed. "release-note/ignore-for-release, release-note/new-feature, release-note/update, release-note/enhancement, release-note/community, release-note/breaking-change, release-note/docs, release-note/infra, release-note/deprecation"
- [x] Accepted the DCO. Commits without the DCO will delay acceptance.
- [x] Made sure tests are passing and test coverage is added if needed.
- [x] Considered the docs impact and opened a new docs issue or PR with docs changes if needed in [website repository](https://github.com/goharbor/website).

### CODE REVIEW

**Review by chlins — APPROVED** [review]
lgtm

File: src/go.mod [hunk 1]
```diff
@@ -21,12 +21,12 @@ require (
 	github.com/go-asn1-ber/asn1-ber v1.5.7
 	github.com/go-ldap/ldap/v3 v3.4.6
 	github.com/go-openapi/errors v0.22.0
-	github.com/go-openapi/loads v0.22.0 // indirect
+	github.com/go-openapi/loads v0.22.0
```
Comment (wy65701436) [inline src/go.mod:24]: Please refrain from submitting code changes that are not related to the pull request.

**Review by wy65701436 — APPROVED** [review]
lgtm

**Review by chlins — APPROVED** [review]
lgtm
